ArticleTemplates/assets/scss/style-sync.scss

```diff
@@ -22,6 +22,7 @@
 @import 'modules/loading';
 @import 'modules/screen-readable';
 @import 'modules/selectable';
+@import 'modules/storyquestions';
```

Comment:
You can just include this in style-async.scss. style-sync.scss is for render-blocking styles generally affecting elements above the fold. style-async.css is added subsequently for below the fold elements, like the story questions.